capstone/src/main/java/com/google/sps/Objects/User.java

```diff
@@ -5,28 +5,31 @@
 public final class User {
 
   private final String userId;
-  private final String name;
-  private final String firstName;
-  private final String lastName;
+  private String name;
+  private String firstName;
+  private String lastName;
   private final String email;
-  private final String phoneNumber;
+  private String phoneNumber;
   private final ArrayList<Badge> badges;
-  private final ArrayList<String> groups;
+  private final ArrayList<Group> groups;
   private final ArrayList<String> interests;
 
   public User(String userId, String firstName, String lastName, String email) {
```

Comment:
That's a good point, at first I had overloaded it so that we could make a User object with limited information, but on second thought we don't actually need both of them.